FirebaseRemoteConfig/Tests/Swift/SwiftAPI/APITestBase.swift

```diff
@@ -88,7 +88,24 @@ class APITestBase: XCTestCase {
         config.configRealtime = RealtimeMocks.mockRealtime(config.configRealtime)
       }
       fakeConsole = FakeConsole()
-      config.configFetch.fetchSession = URLSessionMock(with: fakeConsole)
+      let configuration = URLSessionConfiguration.default
+      configuration.protocolClasses = [MockURLProtocol.self]
+      config.configFetch.fetchSession = URLSession(configuration: configuration)
+
+      var etag = ""
+      MockURLProtocol.requestHandler = { request in
```

Comment:
If you ever need to run some of the tests on watchOS (that don't depend on `URLSession`), I just did this for Vertex AI in the `requestHandler`:
https://github.com/firebase/firebase-ios-sdk/blob/7ea4e9d41b642a455364caf418465f6c8fee52fb/FirebaseVertexAI/Tests/Unit/GenerativeModelTests.swift#L1592-L1596

I see it's `spmbuildonly` and `--skip-tests` on watchOS so no need for now though.

Reply:
Noted, thanks!